cmd/oras/root/pull.go

```diff
@@ -120,6 +124,7 @@ Example - Pull artifact files tagged 'example.com:v1' from an OCI image layout f
 	cmd.Flags().BoolVarP(&opts.KeepOldFiles, "keep-old-files", "k", false, "do not replace existing files when pulling, treat them as errors")
 	cmd.Flags().BoolVarP(&opts.PathTraversal, "allow-path-traversal", "T", false, "allow storing files out of the output directory")
 	cmd.Flags().BoolVarP(&opts.IncludeSubject, "include-subject", "", false, "[Preview] recursively pull the subject of artifacts")
+	cmd.Flags().BoolVarP(&opts.PreservePermissions, "preserve-permissions", "", false, "Preserve original file permissions when extracting tar archives, ignoring the current umask.")
```

Comment:
Should we mark this flag as `[Experimental]` or `[Preview]`?

Since it is already a stable feature in `oras-go`, a `[Preview]` mark should fit.